### ISSUE TITLE
Status of extension on extensions.gnome.org

### ISSUE THREAD

**Issue by matthijskooijman** [projecthamster/shell-extension#304]
I'm a bit confused about the extension at extensions.gnome.org. In particular:
 - I see two identically named extensions, both pointing to this repo: https://extensions.gnome.org/extension/425/project-hamster-extension/ by @elbenfreund and https://extensions.gnome.org/extension/1174/hamster-time-tracker/ by sufiyan. Given that @elbenfreund is a dev here, I suppose that's the primary one?
 - The README states "Current compatible Gnome shell version: 3.28" under "Install For Production", but on extensions.gnome.org it only shows up to shell 3.20. Is the latest released version (0.10.0) perhaps not uploaded to extensions.gnome.org (which *I think* has version 26)?

@elbenfreund, Are you still available to maintain this extension? Is there a need for others to step in?

**Comment by DirkHoffmann** [comment]
Following @elbenfreund's request, here is my audit of the presently four extension providers in https://extensions.gnome.org (aka EGO).

* [elbenfreund](https://extensions.gnome.org/accounts/profile/elbenfreund) repository
   * provides: 3.20,  3.18,  3.16,  3.14,  3.12,  3.10,  3.8,  3.6.1,  3.6,  3.4
* [sufiyan](https://extensions.gnome.org/accounts/profile/sufiyan) repository
   * provides: 3.20, 3.18, 3.16, 3.14, 3.12, 3.10
   * Author is likely to be @muhammadsufiyan.
* [jasuarez](https://extensions.gnome.org/accounts/profile/jasuarez) repository
   * provides: 3.32, 3.34
   * Author is likely to be @jasuarez who forked [hamster-shell-extension](/jasuarez/hamster-shell-extension)
* [lukes](https://extensions.gnome.org/accounts/profile/lukes) repository
   * provides: 3.24, 3.22, 3.20, 3.18. 3.16, 3.14, 3.12, 3.10
   * Named his repo "my extension -- remove it". Should not take long to convince EOG managers to execute his obvious wish.

**Comment by DirkHoffmann** [comment]
@muhammadsufiyan, @jasuarez, if you could answer to this request to clean up (better: remove) your repositories here, it would make this task easier (and more transparent).

**Comment by elbenfreund** [comment]
> Given that @elbenfreund is a dev here, I suppose that's the primary one?

Yes

> Is the latest released version (0.10.0) perhaps not uploaded to extensions.gnome.org

It is, but it has not passed review so far. The "review" - interface was very confusing for me, which even at that time of activity made me realize what is going on far too late.

There are a few minor changes "suggested" by the reviewer, iiuc. I guess we close the review for ``0.10.0`` and focus on the release you guys are working on?

**Comment by mwilck** [comment]
> I guess we close the review for `0.10.0` and focus on the release you guys are working on?

Fine with me, but you may want to comment on the version numbering convention to use in the future (see #319).

**Comment by elbenfreund** [comment]
Btw: Mail about the "please delete me" extension has been send. waiting for feedback.

**Comment by elbenfreund** [comment]
Yuri Konotopov of EGO just got back to me. He will try to get in touch with "lukes" himself in order to see whats what.

We worked together before on deprecated extensions, so I am confident that he will help us out if "lukes" can not be reached in a reasonable timeframe.

**Comment by jasuarez** [comment]
> @MuhammadSufiyan, @jasuarez, if you could answer to this request to clean up (better: remove) your repositories here, it would make this task easier (and more transparent).

Do you mean to remove our extension forks and keep only this one? 

Besides the upgrades for 3.32 and 3.34, I have other features that I need for my own usage (though I think they can be interesting for others too); I made PR some time ago, but probably it should be updated; and I'm working on a new one that I need.

As soon as I finish, I can create a new PR; if it is finally imerged, then I will be more than happier to remove my extension.

**Comment by DirkHoffmann** [comment]
Thanks for your answer, @jasuarez. Yes, we meant that it is too confusing for users to have several extensions with the same name in EGO. 

Good to hear that you want to contribute. But I do not see your request in the [PR list](https://github.com/projecthamster/hamster-shell-extension/pulls?q=is%3Apr+author%3A%40jasuarez). Can you specify what you mean, please?

**Comment by elbenfreund** [comment]
@jasuarez 
Thank you for getting back to us so fast. :)

> Do you mean to remove our extension forks and keep only this one?

We intend to avoid any confusion for regular users about what the "official" extension, supported by us, is. :)

> I made PR some time ago, but probably it should be updated; and I'm working on a new one that I need.

Sounds interesting.

> As soon as I finish, I can create a new PR; if it is finally imerged, then I will be more than happier to remove my extension.

While your extension description leaves no doubt about the fact that this is a personal fork, would you be open to consider renaming your extension title in a way that avoids confusion with official extension?

**Comment by jasuarez** [comment]
> Good to hear that you want to contribute. But I do not see your request in the [PR list]( is:pr author:@jasuarez ). Can you specify what you mean, please?

Because I just closed it :)

As I said, it has been there for a while, and I've been doing changes locally. So I think its better to provide a new MR, which I'll do as soon as I can

**Comment by jasuarez** [comment]
> While your extension description leaves no doubt about the fact that this is a personal fork, would you be open to consider renaming your extension title in a way that avoids confusion with official extension?

Sure! Renamed it to "Colored Hamster Tracker"

**Comment by DirkHoffmann** [comment]
> Because I just closed it :)

Right. My fault. I should have looked [with the correct filter](https://github.com/projecthamster/hamster-shell-extension/pulls?q=is%3Apr+author%3Ajasuarez).

Eager to see your coloured hamster tracker.

**Comment by DirkHoffmann** [comment]
Hi @elbenfreund, 
There are questions on #321 about the status of this cleanup. There are still four extensions on EGO, one of them being @jasuarez's, which may stay there. But the most obvious to be removed is also still there. Do you think this will progress some time soon?
There was also a question how to contact EGO maintainers.
Can you help?

**Comment by DirkHoffmann** [comment]
> Yuri Konotopov of EGO just got back to me. He will try to get in touch with "lukes" himself in order to see whats what.
> 
> We worked together before on deprecated extensions, so I am confident that he will help us out if "lukes" can not be reached in a reasonable timeframe.

Three weeks are almost gone. Do you think it is worth retriggering the admin?

For the records, I contacted @MuhammadSufiyan by email tonight.

**Comment by DirkHoffmann** [comment]
> Can you help?

FTR, answer was given on #304.

**Comment by matthijskooijman** [comment]
Any progress on this issue?

@DirkHoffmann, you said:

> FTR, answer was given on #304.

But that links back to this issue. Did you intend to link elsewhere?

AFAICS, this issue mostly talks about clearing away other uploads of the extension to EGO, but the other part of this issue is uploading new version of the extension to the official repo (or whatever the terminology is) on EGO. AFAIU recent merges have made the extension in this repo usable with newer shell versions, so uploading a new version would make sense? AFAIU this can currently only be done by @elbenfreund? Is there a way this can be distributed to multiple people/accounts? If not, would it make sense to create a "hamster" account on EGO for the official version and upload there? The password could be shared among the maintainers here, or maybe (also) some automatic integration could be set up that autobuilds and uploads releases (AFAIU github can also store secrets and make them available to actions).
